.github/workflows/latest_dependency_checker.yaml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Update dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e .[spark, test]
```

Comment:
Looks like the previous command was `python -m pip install .[spark,test]` without a space between. Would that work here? Might be nice to install them both together as the dependency resolver might arrive at different package versions when we do a separate install - I don't think pip takes into account already installed packages for subsequent install commands.

Reply:
Fixed